horizon/authentication.py

```diff
@@ -1,35 +1,30 @@
+from typing import Annotated
+
 from fastapi import Header, HTTPException, status
 
 from horizon.config import MOCK_API_KEY, sidecar_config
 from horizon.startup.api_keys import get_env_api_key
 
 
-def enforce_pdp_token(authorization=Header(None)):
+def enforce_pdp_token(authorization: Annotated[str | None, Header()]):
     if authorization is None:
-        raise HTTPException(
-            status.HTTP_401_UNAUTHORIZED, detail="Missing Authorization header"
-        )
+        raise HTTPException(status.HTTP_401_UNAUTHORIZED, detail="Missing Authorization header")
     schema, token = authorization.split(" ")
 
     if schema.strip().lower() != "bearer" or token.strip() != get_env_api_key():
         raise HTTPException(status.HTTP_401_UNAUTHORIZED, detail="Invalid PDP token")
 
 
-def enforce_pdp_control_key(authorization=Header(None)):
+def enforce_pdp_control_key(authorization: Annotated[str | None, Header()]):
```

Comment:
are you sure Header() and Header(None) ?

Reply:
https://fastapi.tiangolo.com/tutorial/header-params/?h=header#import-header